docs/import-data.md

```diff
@@ -73,7 +73,7 @@ tool. Next, import the region into the tool:
 $ python source/manage.py add_region <region> <path to shapefile>
 ```
 
-You should also convert the region boundary to GeoJSON and, it to the
+If it doesn't already exist, you should also convert the region boundary to GeoJSON and add it to the
```

Comment:
I added this comment because the file is committed. I still think it's useful to keep these instructions in case regions get updated.